Development/nmos/control_protocol_utils.h

```diff
@@ -0,0 +1,84 @@
+#ifndef NMOS_CONTROL_PROTOCOL_UTILS_H
+#define NMOS_CONTROL_PROTOCOL_UTILS_H
+
+#include "cpprest/basic_utils.h"
+#include "nmos/control_protocol_handlers.h"
+
+namespace nmos
+{
+    struct control_protocol_resource;
+
+    struct control_protocol_exception : std::runtime_error
+    {
+        control_protocol_exception(const std::string& message) : std::runtime_error(message) {}
+    };
+
+    namespace details
+    {
+        // get the runtime property constraints of a given property_id
```

Comment:
```suggestion
        // get the runtime property constraints of a given property
```

Reply:
It is matching to other function's comments too